### ISSUE TITLE
Missing site description 

### ISSUE THREAD

**Issue by dartiss** [WordPress/twentynineteen#110]
The site description is not appearing on individual post or pages, only on the main archive list. However, the title and dash is appearing. 

Here it is being displayed - https://pento.net

Here is it now missing - https://pento.net/2018/10/04/wordpress-5-0-needs-you/

**Comment by enodekciw** [comment]
As far as I understand, hiding it on individual post pages is a design decision ([see this](https://github.com/WordPress/twentynineteen/blob/f0b3e8cdb908aa3cb0a7171dde3c749632c51612/template-parts/header/site-branding.php#L11)).

Strange mdash behaviour should be resolved if #19 is merged

**Comment by dartiss** [comment]
That's... odd 🤔Why would you want a reduced branding on the vast majority of the site?

At the very least I think this should be an option in the customizer

**Comment by patilswapnilv** [comment]
I agree with @dartiss . We can have an option in the customizer. Having description would be required in number of instances.

**Comment by nielslange** [comment]
+1 for @dartiss explanation. This might lead to some dirty hacks to manually add the tagline on other pages.

**Comment by nielslange** [comment]
I created #142 to remove the trailing dash from the site title in case the tagline is empty. This fix is independent of the pages. The trailing dash will also disappear on homepage and blog posts page if no tagline had been provided by the user.

**Comment by designsimply** [comment]
Related: #17 (duplicate even?)

**Comment by nielslange** [comment]
> Related: #17 (duplicate even?)

Yes, looks pretty duplicated to me, @designsimply. I just closed #142 as #129 is a more elegant solution.
